airflow/models.py

```diff
@@ -1744,6 +1749,29 @@ def dry_run(self):
         self.render_templates()
         task_copy.dry_run()
 
+    @provide_session
+    def handle_reschedule(self, reschedule_exception, test_mode=False, context=None,
```

Comment:
docstring on the function

Reply:
or is it a private function / method which name may change to _handle_reschedule?

Reply:
Yes, should be private